autogen/main/cluster.tf.tmpl

```diff
@@ -306,6 +306,13 @@ resource "google_container_cluster" "primary" {
       security_group = authenticator_groups_config.value.security_group
     }
   }
+
+  notification_config {
+    pubsub {
+      enabled = var.enable_notification_config
```

Comment:
Can we do this with a single var? I was thinking we could just utilize `notification_config_topic` like 
```hcl
pubsub{
enabled = var.notification_config_topic != "" ? true : false
topic = var.notification_config_topic
}
```

Reply:
Absolutely. But technically it is possible to set the topic reference but disable the update itself.

Reply:
I don't see an apparent use case that needs disabled but with `notification_config_topic` set. Happy to revisit if needed.